lib/sign_notarize.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+set -ex
+
+if [[ -z $WORKSPACE ]]; then
+  WORKSPACE=$PWD
+fi
+
+UNAME=`uname`
+
+cd "${WORKSPACE}/SapMachine"
+
+echo "Git Revision=$(git rev-parse HEAD)"
+
+cd build
+cd "$(ls)"
+cd bundles
+
+JDK_NAME=$(ls *jdk-*_bin.*) || true
+if [ -z $JDK_NAME ]; then
+  JDK_NAME=$(ls *jdk-*_bin-debug.*)
+fi
+if [[ $JDK_NAME = sapmachine-* ]]; then
+  SAPMACHINE_BUNDLE_PREFIX="sapmachine-"
+fi
+read JDK_VERSION JDK_SUFFIX<<< $(echo $JDK_NAME | sed $SEDFLAGS 's/'"${SAPMACHINE_BUNDLE_PREFIX}"'jdk-([0-9]+((\.[0-9]+))*)(.*)/ \1 \4 /p')
+JDK_BUNDLE_NAME="${SAPMACHINE_BUNDLE_PREFIX}jdk-${JDK_VERSION}${JDK_SUFFIX}"
+JRE_BUNDLE_NAME="${SAPMACHINE_BUNDLE_PREFIX}jre-${JDK_VERSION}${JDK_SUFFIX}"
+SYMBOLS_BUNDLE_NAME=$(ls *_bin-*symbols.*)
+
+if [ "$RELEASE" == true ]; then
+  # remove build number +xx from release build filenames
+  ARCHIVE_NAME_JDK="$(echo $JDK_BUNDLE_NAME | sed 's/\+[0-9]*//')"
+  ARCHIVE_NAME_JRE="$(echo $JRE_BUNDLE_NAME | sed 's/\+[0-9]*//')"
+  ARCHIVE_NAME_SYMBOLS="$(echo $SYMBOLS_BUNDLE_NAME | sed 's/\+[0-9]*//')"
+else
+  # substitute build number +xx to .xx to avoid problems with uploads. + is no good character :-)
+  ARCHIVE_NAME_JDK="$(echo $JDK_BUNDLE_NAME | sed 's/\+/\./')"
+  ARCHIVE_NAME_JRE="$(echo $JRE_BUNDLE_NAME | sed 's/\+/\./')"
+  ARCHIVE_NAME_SYMBOLS="$(echo $SYMBOLS_BUNDLE_NAME | sed 's/\+/\./')"
+fi
+
+if [[ $UNAME == Darwin ]]; then
```

Comment:
this if condition is not needed. Script is run on Darwin only, anyway (by template).

Reply:
done